src/main/java/io/thinkit/edc/client/connector/services/VerifiableCredentials.java

```diff
@@ -0,0 +1,175 @@
+package io.thinkit.edc.client.connector.services;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.thinkit.edc.client.connector.model.Result;
+import io.thinkit.edc.client.connector.model.VerifiableCredentialManifest;
+import io.thinkit.edc.client.connector.model.VerifiableCredentialResource;
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+import java.util.function.UnaryOperator;
+
+public class VerifiableCredentials {
+    private final String url;
+    private final EdcApiHttpClient edcApiHttpClient;
+
+    private final ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
inject, there should be a single `ObjectMapper` instance in the whole client (because it should be injectable also from the client application point of view eventually)